### PR TITLE
Rancher server chart

### PR DESCRIPTION
**Pull request by jgreat**
**What this PR does / why we need it**:

Add a chart for installing Rancher Server. Rancher is a management platform for kubernetes clusters.

https://rancher.com

Details for the install are in the included README.md 

**Special notes for your reviewer**:

I've tried to follow the best practices and review guides. Please let me know if I missed anything.

### CODE REVIEW

**Comment by jgreat** [comment]
cla should be signed now.

File: stable/rancher/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+approvers:
+  - jgreat
+  - ibuildthecloud
+  - cloudnautique
+  - deniseschannon
+  - vincent99
```
Comment (rjkernick) [inline stable/rancher/OWNERS:6]: All approvers need to be members of the kubernetes org on github. Please see:  https://github.com/kubernetes/community/blob/master/community-membership.md

File: stable/rancher/templates/certificate.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{{- if eq .Values.ingress.tls "letsEncrypt" -}}
```
Comment (rjkernick) [inline stable/rancher/templates/certificate.yaml:1]: Why not take advantage of cert manager's ingress-shim like most other charts do?

Comment (jgreat) [reply stable/rancher/templates/certificate.yaml:1]: I found the cert-manager docs a little vague.  I thought the shim was the temporary endpoint that gets set up to return the challenge for LE and part of the cert-manager chart?  

If there is a better way, I'm all for it.  Can you point me at an example?

File: stable/rancher/OWNERS (hunk 1 above)
Comment (jgreat) [reply stable/rancher/OWNERS:6]: I guess I'm uncertain on how we would achieve this. We (Rancher Labs) would really like to have a chart to install Rancher in the main public helm repo, manage and maintain this chart.

How do other companies handle this?

Comment (rjkernick) [reply stable/rancher/OWNERS:6]: I am not sure. @unguiculus might have a better answer

**Comment by jgreat** [comment]
I'm going to close this request for now.  I think we are going to start out by hosting our own chart. Thanks for the feedback.

**Pull request closed without merging**